parsl/executors/workqueue/executor.py

```diff
@@ -136,40 +140,44 @@ def WorkQueueSubmitThread(task_queue=multiprocessing.Queue(),
                 remapping_string = "-r " + remapping_string
                 remapping_string = remapping_string[:-1]
 
+            # Create command string
             logger.debug(launch_cmd)
             command_str = launch_cmd.format(input_file=function_data_loc_remote,
                                             output_file=function_result_loc_remote,
                                             remapping_string=remapping_string)
-
-            logger.debug(command_str)
             command_str = std_string + command_str
             logger.debug(command_str)
 
+            # Create WorkQueue task for the command
             logger.debug("Sending task {} with command: {}".format(parsl_id, command_str))
             try:
                 t = Task(command_str)
             except Exception as e:
                 logger.error("Unable to create task: {}".format(e))
                 continue
+
+            # Specify environment variables for the task
             if env is not None:
                 for var in env:
                     t.specify_environment_variable(var, env[var])
 
+            # Specify script, and data/result files for task
             t.specify_file(full_script_name, script_name, WORK_QUEUE_INPUT, cache=True)
-            t.specify_file(function_result_loc, function_result_loc_remote, WORK_QUEUE_OUTPUT, cache=False)
             t.specify_file(function_data_loc, function_data_loc_remote, WORK_QUEUE_INPUT, cache=False)
+            t.specify_file(function_result_loc, function_result_loc_remote, WORK_QUEUE_OUTPUT, cache=False)
             t.specify_tag(str(parsl_id))
+            logger.debug(t.id)
```

Comment:
it can be frustrating to get a value appearing in the log message without a description like "here is the task id: {}"